src/DotNetty.Handlers/Tls/TlsHandler.cs

```diff
@@ -718,7 +711,6 @@ public void ExpandSource(int count)
                 }
                 this.sslOwnedBuffer = default(ArraySegment<byte>);
 
-#if NETSTANDARD1_3
```

Comment:
SslStream API necessary for efficient interaction in synchronous context are present in .net standard 2.0 (unlike in 1.3). So, for TlsHandler.cs specifically, we should use `#else` branch instead now.